project2-beat-that/script.js

```diff
@@ -1,4 +1,124 @@
+// function to roll random dice
+var rollDice = function () {
+  // return 5; - testing draw results
+  var randomDecimal = Math.random() * 6;
+  var randomInteger = Math.floor(randomDecimal);
+  var diceNumber = randomInteger + 1;
+  return diceNumber;
+};
+
+// global variables
+// initial game mode
+var gameMode = "playerOneRoll";
+// arrays to store the dice numbers rolled
+var playerOneNumbers = [];
+var playerTwoNumbers = [];
+// for the player combined numbers
+var playerOneScore;
+var playerTwoScore;
+// player accumulated score
+//var totalPlayerOneScore;
+//var totalPlayerTwoScore;
+
+// game results helper function - can it be here??
+var gameResults = function () {
+  var winnerMessage = `Player One's Number: ${playerOneScore} <br> Player Two's Number: ${playerTwoScore}`;
+  var winnerScore = Math.min(playerOneScore, playerTwoScore);
+  console.log(playerOneScore, playerTwoScore, winnerScore);
+  if (playerOneScore == playerTwoScore) {
+    winnerMessage += "<br> It's a draw!";
+  } else if (playerOneScore == winnerScore) {
+    winnerMessage += "<br> Player 2 Wins! 🏆";
+  } else winnerMessage += "<br> Player 1 Wins! 🏆";
+  return winnerMessage;
+};
```

Comment:
Yes this helper function can be here, and nice job on refactoring the results game mode! Good use of `Math.min` here :)